understanding/pause-stop-hide.html

```diff
@@ -124,101 +114,76 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
-                        ability to use the whole page, all content on the Web page (whether it is used to
-                        meet other success criteria or not) must meet this success criterion. See <a href="#cc5">Conformance Requirement 5: Non-Interference</a>.
+                     <p class="note-title marker">注記</p>
+                     <p>この達成基準を満たさないコンテンツでは、利用者がそのウェブページ全体を使用できない恐れがあるため、ウェブページ上のすべてのコンテンツは他の達成基準を満たすために用いられているか否かにかかわらず、この達成基準を満たさなければならない。<a href="https://waic.jp/translations/WCAG22/#cc5">適合要件 5: 非干渉</a>を参照。
                         
                      </p>
                   </div>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Content that is updated periodically by software or that is streamed to the user agent
-                        is not required to preserve or present information that is generated or received between
-                        the initiation of the pause and resuming presentation, as this may not be technically
-                        possible, and in many situations could be misleading to do so.
+                     <p class="note-title marker">注記</p>
+                     <p>定期的にソフトウェアによって自動的に更新されるコンテンツ、又はユーザエージェントにストリーム配信されるコンテンツでは、コンテンツ再生の一時停止と再開の操作の間に生成、又は受信される情報を保持、又は提示する必要はない。これは技術的に不可能であることが考えられ、多くの状況において利用者の混乱を招くことにつながる可能性があるためである。
                         
                      </p>
                   </div>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>An animation that occurs as part of a preload phase or similar situation can be considered
-                        essential if interaction cannot occur during that phase for all users and if not indicating
-                        progress could confuse users or cause them to think that content was frozen or broken.
+                     <p class="note-title marker">注記</p>
+                     <p>コンテンツの読み込み中やそれに類似した状況の一部として表示されるアニメーションについては、この段階ですべての利用者に対していかなるインタラクションも発生する可能性がなく、かつコンテンツ読み込みの進行状況を表示しないことが利用者の混乱を招いたり、コンテンツが動作を停止した、又はコンテンツが破損しているという誤解を生じたりする可能性がある場合には、必要不可欠なものと考えることができる。
                         
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Fewer users are distracted by content that updates or moves.</dd>
+                  <dt>目標</dt>
+                  <dd>更新又は動きのあるコンテンツに注意をそらされる利用者を少なくする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Let users control content changes that occur in parallel with other content.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>他のコンテンツと並行して発生するコンテンツの変更を、利用者に制御させる。</dd>
```

Comment:
ここでのoccurは「存在する」とするのが自然でしょうか。

```suggestion
                  <dd>他のコンテンツと並行して存在するコンテンツの変更を、利用者に制御させる。</dd>
```

Reply:
たしかに、「存在する」のほうが自然でよいと思います！